docs/setup-vue-app.md

```diff
@@ -51,7 +51,7 @@ npm run build
 #### Unit tests
 
 ```sh
-npm run test:unit
+npm run test
```

Comment:
The distinction is there to make it different from the e2e tests, so having the `:unit` suffix feels a little more appropriate to me.

Reply:
Then we need to add it to the CMS as well?

Reply:
Added the full command to the documentation for clarity